From 5313a314bb7a8c65007adf80e76ee507817e6667 Mon Sep 17 00:00:00 2001
From: Meticulus <theonejohnnyd@gmail.com>
Date: Thu, 23 Nov 2017 08:22:17 -0600
Subject: [PATCH] "0001-BatteryService-hack-fix-battery-led
 _______________________________

*Previous From 2209f7b68ed3cef03afb0db442b92d0536268c7b Mon Sep 17 00:00:00 2001
*Previous From: Meticulus <theonejohnnyd@gmail.com>
*Previous Date: Wed, 14 Jun 2017 13:57:44 -0500
*Previous Subject: [PATCH] BatteryService: hack fix battery led
*
*Change-Id: I00459a667de964ffe5392dd2170e57a7eaef0b7d"

Change-Id: I6667682934410c9065244fcfe3c44e1e5f01a90f
---
 .../java/com/android/server/BatteryService.java    | 31 +++++++---------------
 1 file changed, 9 insertions(+), 22 deletions(-)

diff --git a/services/core/java/com/android/server/BatteryService.java b/services/core/java/com/android/server/BatteryService.java
index 25ed5de..6190601 100644
--- a/services/core/java/com/android/server/BatteryService.java
+++ b/services/core/java/com/android/server/BatteryService.java
@@ -1050,38 +1050,25 @@ public final class BatteryService extends SystemService {
             Resources res = mContext.getResources();
 
             // Battery light enabled
-            mLightEnabled = Settings.System.getIntForUser(resolver,
-                    Settings.System.BATTERY_LIGHT_ENABLED, mLightEnabled ? 1 : 0, UserHandle.USER_CURRENT) != 0;
+            mLightEnabled = true;
 
             //Fast charging LED enabled
-            mFastBatteryLightEnabled = Settings.System.getIntForUser(resolver,
-                    Settings.System.FAST_CHARGING_LED_ENABLED, 0, UserHandle.USER_CURRENT) != 0;
+            mFastBatteryLightEnabled = true;
 
             // Low battery pulse
-            mLedPulseEnabled = Settings.System.getIntForUser(resolver,
-                    Settings.System.BATTERY_LIGHT_PULSE, mLightEnabled ? 1 : 0, UserHandle.USER_CURRENT) != 0;
+            mLedPulseEnabled = true;
 
             // Only light when fully charged
-            mLightOnlyFullyCharged = Settings.System.getIntForUser(resolver,
-                    Settings.System.BATTERY_LIGHT_ONLY_FULLY_CHARGED, 0, UserHandle.USER_CURRENT) != 0;
+            mLightOnlyFullyCharged = true;
 
             // Light colors
-            mBatteryLowARGB = Settings.System.getIntForUser(resolver,
-                    Settings.System.BATTERY_LIGHT_LOW_COLOR,
-                    res.getInteger(com.android.internal.R.integer.config_notificationsBatteryLowARGB), UserHandle.USER_CURRENT);
-            mBatteryMediumARGB = Settings.System.getIntForUser(resolver,
-                    Settings.System.BATTERY_LIGHT_MEDIUM_COLOR,
-                    res.getInteger(com.android.internal.R.integer.config_notificationsBatteryMediumARGB), UserHandle.USER_CURRENT);
-            mBatteryFullARGB = Settings.System.getIntForUser(resolver,
-                    Settings.System.BATTERY_LIGHT_FULL_COLOR,
-                    res.getInteger(com.android.internal.R.integer.config_notificationsBatteryFullARGB), UserHandle.USER_CURRENT);
-            mBatteryReallyFullARGB = Settings.System.getIntForUser(resolver,
-                    Settings.System.BATTERY_LIGHT_REALLY_FULL_COLOR, mBatteryFullARGB, UserHandle.USER_CURRENT);
+            mBatteryLowARGB = 0xffff0000;
+            mBatteryMediumARGB = 0xffffff00;
+            mBatteryFullARGB = 0xff00ff00;
+            mBatteryReallyFullARGB = 0xff0000ff;
 
             // Fast Light colors
-            mFastBatteryARGB = Settings.System.getIntForUser(resolver,
-                    Settings.System.FAST_BATTERY_LIGHT_COLOR,
-                    res.getInteger(com.android.internal.R.integer.config_notificationsFastBatteryARGB), UserHandle.USER_CURRENT);
+            mFastBatteryARGB = 0xffffffff;
 
             updateLedPulse();
         }
-- 
2.7.4

